stable/openvpn/README.md

```diff
@@ -23,44 +23,54 @@ POD_NAME=$(kubectl get pods -l type=openvpn -o jsonpath='{.items[0].metadata.nam
 
 When all components of the openvpn chart have started use the following script to generate a client key:
 
-```bash
-#!/bin/bash
-
-if [ $# -ne 1 ]
-then
-  echo "Usage: $0 <CLIENT_KEY_NAME>"
-  exit
-fi
-
-KEY_NAME=$1
-NAMESPACE=$(kubectl get pods --all-namespaces -l type=openvpn -o jsonpath='{.items[0].metadata.namespace}')
-POD_NAME=$(kubectl get pods -n $NAMESPACE -l type=openvpn -o jsonpath='{.items[0].metadata.name}')
-SERVICE_NAME=$(kubectl get svc -n $NAMESPACE -l type=openvpn  -o jsonpath='{.items[0].metadata.name}')
-SERVICE_IP=$(kubectl get svc -n $NAMESPACE $SERVICE_NAME -o go-template='{{range $k, $v := (index .status.loadBalancer.ingress 0)}}{{$v}}{{end}}')
-kubectl -n $NAMESPACE exec -it $POD_NAME /etc/openvpn/setup/newClientCert.sh $KEY_NAME $SERVICE_IP
-kubectl -n $NAMESPACE exec -it $POD_NAME cat /etc/openvpn/certs/pki/$KEY_NAME.ovpn > $KEY_NAME.ovpn
+```shell
+kubectl -n <namespace> exec -it <pod_name> /etc/openvpn/setup/newClientCert.sh <key_name> <external-ip>
+kubectl -n <namespace> exec -it <pod_name> /etc/openvpn/setup/newClientCert.sh <key_name> > <key_name>.ovpn
 ```
 
+The entire list of helper scripts can be found on [templates/config-openvpn.yaml](templates/config-openvpn.yaml)
```

Comment:
I understand the comment, but it would be really nice if there was a simple list of the available scripts and their purpose and default usage in the README as well. In general, I'd prefer it if users are not required to read the actual templates to understand functionality.